installation/scripts/generate-kyma-installer.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+KYMA_PATH="${CURRENT_DIR}/../.."
+INSTALLER_YAML_PATH="${KYMA_PATH}/installation/resources/installer.yaml"
+UI_TEST_SECRET_PATH="${KYMA_PATH}/installation/resources/ui-test-secret.yaml.tpl"
+
+VERSIONS_ENV_PATH=""
+INSTALLER_CONFIG=""
+OUTPUT_PATH=""
+
+if [[ ! $# -eq 2 ]] ; then
+    echo 'Provide correct input arguments'
+    exit 1
+fi
+
+case "$1" in
+    "local")   
+        INSTALLER_CONFIG="${KYMA_PATH}/installation/resources/installer-config-local.yaml.tpl"
```

Comment:
i'm not sure wether we want to have the file hardcoded. Having path to the file as parameter would be more flexible i guess but then condition about ui test secret might not be possible